src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,131 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: $card-gap-vertical $card-gap-horizontal;
+  padding: $catalog-padding-vertical $catalog-padding-horizontal;
+
+  @media (min-width: 488px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+}
+
+.card {
+  background: $color-bg-white;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  position: relative;
+  border-radius: 8px;
```

Comment:
The 'border-radius' property is defined twice for the '.card' class. Consider removing the duplicate declaration to avoid confusion and ensure consistency.